multicluster/controllers/multicluster/stale_controller_test.go

```diff
@@ -655,16 +655,17 @@ func TestStaleController_CleanupLabelIdentites(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			stopCh := make(chan struct{})
+			defer close(stopCh)
 			fakeClient := fake.NewClientBuilder().WithScheme(scheme).WithLists(tt.existLabelIdentityList).Build()
 			fakeRemoteClient := fake.NewClientBuilder().WithScheme(scheme).WithLists(tt.existingResImpList).Build()
 			ca := commonarea.NewFakeRemoteCommonArea(fakeRemoteClient, "leader-cluster", localClusterID, "antrea-mcs")
 
 			mcReconciler := NewMemberClusterSetReconciler(fakeClient, scheme, "default")
 			mcReconciler.SetRemoteCommonArea(ca)
 			c := NewStaleResCleanupController(fakeClient, scheme, "default", mcReconciler, MemberCluster)
-			if err := c.cleanup(); err != nil {
-				t.Errorf("StaleController.cleanup() should clean up all stale LabelIdentities but got err = %v", err)
-			}
+			go c.Run(stopCh)
+			time.Sleep(100 * time.Millisecond)
```

Comment:
Why do you remove the line to call `c.cleanup()`? what's the point here without calling the function?

Reply:
this is only for cover the c.Run() part here, and at the last it should also have the same results to the c.cleanup()